packages/container/src/layout.tsx

```diff
@@ -8,13 +8,11 @@ const titleCase = (title: string) => title.split('-').map((word) => capitalizeFi
 const Header: React.FunctionComponent = () => (
   <nav className="flex p-4 bg-indigo-400 text-indigo-100">
     {['home', ...microfrontends].map((mf) => {
-      const { pathname } = window.location;
       const path = `/frontend-v2/${mf === 'home' ? '' : mf}`;
-      const active = mf === 'home' ? pathname === '/frontend-v2/' : pathname.includes(mf);
       return (
-        <a className={cx({ 'text-white': active }, 'hover:text-white mr-4')} key={mf} href={path}>
+        <Link className="hover:text-white mr-4" key={mf} to={path}>
```

Comment:
Also, using the `HashRouter` rather than `BrowserRouter` would make it possible to access the path `/frontend-v2/party-reputation` directly without first having to visit the homepage and click on the link. Otherwise, we would get the 404